.cicd/platforms/pinned/centos-8-pinned.dockerfile

```diff
@@ -10,8 +10,7 @@ RUN yum update -y && \
     yum install -y python3 glibc-locale-source glibc-langpack-en && \
     yum clean all && rm -rf /var/cache/yum
 RUN dnf install -y  https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm && \
-    dnf group install -y  "Development Tools" && \
-    dnf --enablerepo=powertools install -y doxygen ocaml
```

Comment:
Instead of wholesale removing can this be fixed? Otherwise why bother installing this on _any_ platform?

Reply:
PowerTool repo has been move to a subscription based service, we need subscription to access to repo, I didn't check the pricing though. I just remove it to see if any downstream CI steps requires it and it doesn't seem to be because the entire pipeline succeeded without issue.

Reply:
I just did a `dnf --enablerepo=powertools install -y doxygen ocaml` on a fresh centos8 install and it worked. The problem was likely due to unpinned still using `PowerTools` instead of `powertools`